init-tools.msbuild

```diff
@@ -1,14 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
+  <!-- BaseIntermediateOutputPath must be set before Microsoft.Common.props is loaded. --> 
+  <PropertyGroup>
+    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)Tools/$(BuildToolsPackageVersion)/</BaseIntermediateOutputPath>
+  </PropertyGroup>
+
+  <Import Project="Directory.Build.props" />
```

Comment:
Why do you need to explicitly import `Directory.Build.props`?  Just importing the Sdk.props will do this for you.

Reply:
good catch, need to check.